luckystrikehoreca/app/Http/Controllers/CateringController.php

```diff
@@ -20,57 +20,73 @@ public function getCateringItems()
         $cateringItems = CateringItem::all();
         // Fetch the unique_identifier from the store you get the second you enter the site
         $uniqueIdentifier = session('unique_identifier');
-        
         // Get the database object with this number and the date of today
         $userConnect = LoginUserConnect::where('unique_identifier', $uniqueIdentifier)
-        ->orderBy('date', 'desc')
-        ->first();
+            ->orderBy('date', 'desc')
```

Comment:
instead of ->orderBy('date', 'desc') you can use ->latest('date')

not neccesary for a merge just a tip for the future